pkgs/development/python-modules/sslyze/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, callPackage, fetchPypi, fetchFromGitHub, buildPythonPackage, nassl, cryptography, cryptography_vectors, typing-extensions, faker }:
```

unused imports, white space, and generally for lonog lists of inputs, we do 1 per line to help with parsing later diffs
```suggestion
{ lib, fetchPypi, fetchFromGitHub, buildPythonPackage
, nassl
, cryptography
, cryptography_vectors
, typing-extensions
, faker
}:

```